python/pyspark/sql/readwriter.py

```diff
@@ -367,16 +370,19 @@ def format(self, source):
     @since(1.5)
     def option(self, key, value):
         """Adds an output option for the underlying data source.
+
+        >>> csvpath = os.path.join(tempfile.mkdtemp(), 'data')
+        >>> df.write.option('quote', None).format('csv').save(csvpath)
```

Comment:
What does this `None` mean? It has the same behavior than without `option('quote', None)`, right?

This is an confusing example, can we move it to unit test?